server/routes/offenceCodeDecisions/decisionForm.ts

```diff
@@ -0,0 +1,20 @@
+// Form data backing a decision. Used in form submits and to allow the saving of form data on a decision page so that
+// we can repopulate it when returning from a user lookup.
+export type DecisionForm = {
+  selectedDecisionId?: string
+  selectedDecisionData?: StaffData | OfficerData | PrisonerData
```

Comment:
The additional data can currently be one of three (actually really only 2 at the moment) types. (There will probably be one more - for the another person)